src/test/java/stepDefinition/GenericStepDef.java

```diff
@@ -38,6 +38,6 @@ public void userPopulatesFieldsWithValues(Map<String, String> data) {
     public void userIsSuccessfullyLoggedIn(String elementName, String extension) {
         waitForPageLoaded();
         waitVisibility(getElementByName(elementName.concat(extension)));
-        Assert.assertTrue(String.format("%s element is displayed", elementName), getElementByName(elementName.concat(extension)).isDisplayed());
+        assertTrue(String.format("%s element is displayed", elementName), getElementByName(elementName.concat(extension)).isDisplayed());
```

Comment:
assert message has the same problem could you check them all

Reply:
Don't use such long line

Reply:
done, thank you !